api/graphql/schema.graphql

```diff
@@ -99,8 +106,16 @@ type Membership {
   lastViewedAt: String
   newPostCount: Int
   community: Community
+  settings: MembershipSettings
 }
 
+type MembershipSettings {
+  send_push_notifications: Boolean
```

Comment:
Hmm. Basically what it would buy us is consistency: we'd be able to rely on the rule "All attributes in the API and frontend are camel case." It's pretty straightforward to use lodash.snakeCase to convert back inside the backend.